packages/gatsby/src/db/__tests__/nodes.js

```diff
@@ -336,14 +324,17 @@ describe(`nodes db tests`, () => {
     )
     expect(getNode(`hi`)).toMatchObject({ id: `hi` })
     store.dispatch(
-      actions.deleteNode(`hi`, getNode(`hi`), {
-        name: `tests`,
-      })
+      actions.deleteNode(
+        { node: getNode(`hi`) },
+        {
+          name: `tests`,
+        }
+      )
     )
     expect(getNode(`hi`)).toBeUndefined()
     const deprecationNotice =
-      `Calling "deleteNode" with a nodeId is deprecated. Please pass an ` +
-      `object containing a full node instead: deleteNode({ node }). ` +
+      `Calling "deleteNode" with an object containing a full node is deprecated. Please pass ` +
```

Comment:
I might change this slightly?

"Calling 'deleteNode' with {node} is deprecated. Please pass the node directly to the function: deleteNode(node)."

An object containing the full node reads a bit confusing in my mind.